ecommerce_api_client/auth.py

```diff
@@ -1,11 +1,16 @@
+import datetime
+
 import jwt
+
```

Comment:
There shouldn't be a newline here; both `jwt` and `requests` are third-party modules.